extension/src/popup/ducks/transactionSubmission.ts

```diff
@@ -580,7 +582,7 @@ export const initialState: InitialState = {
   destinationBalances: {
     tokensWithNoBalance: [],
     balances: null,
-    isFunded: false,
+    isFunded: true,
```

Comment:
default to true because the negative case shows an error. Let's not show it until we're sure

Reply:
I've been thinking that `isFunded` should actually be `boolean | null` because we've run into several unrelated bugs where the default boolean is incorrect due to an error in trying to populate it. Not recommending doing that now but wanted to surface for discussion.